README.md

```diff
@@ -88,3 +89,31 @@ gswt() {
   cd "$(_fzf_git_worktrees --no-multi)"
 }
 ```
+
+Note for [zsh-vi-mode] users
+----------------------------
+
+The built-in vi mode for zsh should work fine, but if you use the [zsh-vi-mode]
+plugin for ZSH then you will need to add the following to your `.zshrc` file
+in order to use the key bindings:
+
+```sh
+# Set key bindings for zsh-vi-mode insert mode.
+function zvm_after_init() {
+  zvm_bindkey viins "^P" up-line-or-beginning-search
+  zvm_bindkey viins "^N" down-line-or-beginning-search
+  for o in files branches tags remotes hashes stashes lreflogs each_ref; do
```

Comment:
This is missing the recent addition of "worktrees". Whenever we add something, each user will have to update their configuration accordingly not to miss the update. How about providing a function instead so that the users don't have to manually update their code?

```sh
# Call _fzf_git_zvm_after_init from your zvm_after_init
function zvm_after_init() {
  _fzf_git_zvm_after_init
}